src/main/java/IO/IO.java

```diff
@@ -0,0 +1,78 @@
+package IO;
+
+import com.github.javafaker.Faker;
+import org.apache.poi.hssf.usermodel.HSSFCell;
+import org.apache.poi.hssf.usermodel.HSSFRow;
+import org.apache.poi.hssf.usermodel.HSSFSheet;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class IO {
+
+    private Faker faker = new Faker();
+
+    private void exists(String fileName) throws IOException {
```

Comment:
poor naming, I would expect return type of boolean from method with name "exists"
this method looks like redundant and can be avoided by 
`if (file = null) {
file.createNewFile()
}`

Reply:
fix